bobikApp/Person.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Person.swift
+//  bobikApp
+//
+//  Created by Vasyl Denys on 1/21/24.
+//
+
+import Foundation
+
+class Person {
```

Comment:
It is preferable to use structures rather than classes. The latter are best used only if you want to take advantage of the reference type.

Reply:
done